nipype/interfaces/base/traits_extension.py

```diff
@@ -304,6 +305,11 @@ def validate(self, objekt, name, value, return_pathlike=False):
         return value
 
 
+# Patch in traits these two new
+traits.File = File
+traits.Directory = Directory
```

Comment:
I think there's a trade-off between breaking compatibility with other software (via traits) and the likelihood of a user mistakenly using `traits.File` instead of `File`.

Happy not to patch, although I don't see it as risky.

Reply:
I would prefer not to patch, as we've been using these traits without doing so for years now.

Reply:
true for File, not always the case for Directory. I actually patched because I fell into this myself when writing a test.

Reply:
however, since this is controversial I'll leave the patching out of this PR and we may consider it again in the future